website/docs/feature-flag-tutorials/python/implementing-feature-flags.md

```diff
@@ -209,14 +213,24 @@ This will require us to:
 - Create a delete button
 - Map the delete button to the delete method
 
-In `src/backend/surveys/routes.py`, add `client` to the existing `backend` import statement. The full import line will now look like this:
+First, we need an error handler to return a simple 404 page to stop a user from being able to delete a survey when the flag is off. We will use this function in our delete method.
+
+In your `routes.py` file, import a module from Flask that will support error handling: [`abort`](https://flask.palletsprojects.com/en/3.0.x/api/#flask.abort).
+
+Line 1 will now look like this:
+
+```py
+from flask import redirect, render_template, request, url_for, abort
+```
+
+Add `client` to the `backend` import statement on line 4. The full import line will now look like this:
 
 
 ```py
 from backend import db, client
 ```
 
-We’ve imported the initialized Unleash client into `routes.py`. Now we can use that data to pass into the `surveys_list_page` method. This will allow us to check the status of the enabled flag to conditionally render the 'Delete' button on the surveys page.
+We’ve imported the initialized Unleash client into `routes.py`. Now we can use that data to pass into the `surveys_list_page` method. This will allow us to check the status of the enabled flag to conditionally render the delete button on the surveys HTML page.
```

Comment:
Personally I would just leave HTML out. Hopefully anyone reading this knows what HTML is at this point. Entirely up to you